src/sage/rings/function_field/function_field.py

```diff
@@ -929,50 +938,52 @@ def valuation(self, prime):
         applying the substitution `x \mapsto 1/x` (here, the inverse map is also `x
         \mapsto 1/x`)::
 
-            sage: w = valuations.GaussValuation(R, QQ.valuation(2)).augmentation(x, 1)  # optional - sage.rings.function_field
-            sage: w = K.valuation(w)                                                    # optional - sage.rings.function_field
-            sage: v = K.valuation((w, K.hom([~K.gen()]), K.hom([~K.gen()]))); v         # optional - sage.rings.function_field
+            sage: w = valuations.GaussValuation(R, QQ.valuation(2)).augmentation(x, 1)  # needs sage.rings.function_field
+            sage: w = K.valuation(w)                                                    # needs sage.rings.function_field
+            sage: v = K.valuation((w, K.hom([~K.gen()]), K.hom([~K.gen()]))); v         # needs sage.rings.function_field
             Valuation on rational function field
             induced by [ Gauss valuation induced by 2-adic valuation, v(x) = 1 ]
             (in Rational function field in x over Rational Field after x |--> 1/x)
 
         Note that classical valuations at finite places or the infinite place are
         always normalized such that the uniformizing element has valuation 1::
 
-            sage: K.<t> = FunctionField(GF(3))                                          # optional - sage.rings.finite_rings sage.rings.function_field
-            sage: M.<x> = FunctionField(K)                                              # optional - sage.rings.finite_rings sage.rings.function_field
-            sage: v = M.valuation(x^3 - t)                                              # optional - sage.rings.finite_rings sage.rings.function_field
-            sage: v(x^3 - t)                                                            # optional - sage.rings.finite_rings sage.rings.function_field
+            sage: # needs sage.rings.finite_rings sage.rings.function_field
```

Comment:
This is the first time I see `needs sage.rings.function_field` for a call to `FunctionField`. It should be the same everywhere.